src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterToTypeResolverTests.cs

```diff
@@ -18,22 +18,27 @@ public class SearchParameterToTypeResolverTests
         private static readonly FhirPathCompiler _compiler = new FhirPathCompiler();
 
         [Fact]
-        public void GivenAFhirPathExpressionWithFirstFunction_WhenResolvingTypes_ThenTheyAreRetunedCorrectly()
+        public void GivenAFhirPathExpressionWithFirstFunction_WhenResolvingTypes_ThenTheyAreReturnedCorrectly()
         {
             var expression = _compiler.Parse("Patient.extension.where(url = 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-race').first().extension.where(url = 'ombCategory').value");
             SearchParameterTypeResult[] results = SearchParameterToTypeResolver.Resolve(
                 KnownResourceTypes.Patient,
                 (SearchParamType.Token, expression, new Uri("http://hl7.org/fhir/SearchParameter/Patient-race")),
                 null).ToArray();
 
-            if (ModelInfoProvider.Version == FhirSpecification.Stu3)
-            {
-                Assert.Equal(38, results?.Length);
-            }
-            else
-            {
-                Assert.Equal(49, results?.Length);
-            }
+            Assert.Equal("DataType", results.Single().FhirNodeType);
```

Comment:
@Ivanidzo4ka @feordin test how/if this changes the SearchParameter support/detection when posting in an extension like this

Reply:
yeah, it's broken.

Looking into it...